css/style.css

```diff
@@ -144,21 +168,30 @@ a:active {
   max-width: 960px;
 }
 
+.primary-content2 {
+  width: 75%;
+  padding-left: 25px;
+  padding-right: 25px;
+  margin: auto;
+  max-width: 960px;
+}
+
 .approach {
   color: #878787;
 }
 
 .whySAM {
   color: white;
   text-align: left;
-  padding: 18% 10%;
+  padding: 18% 10% 18% 10%;
```

Comment:
the padding is pushing your text away from the edges of the pic. if you want the text to sit at the very top of the image it'd be:

`padding: 0 10%;`